### PR TITLE
PR for isue #5578 "implement CyclingOutputContainerLayoutManager" -- JS impl

### PR DESCRIPTION
**Pull request by konstantinTarletski**

### CODE REVIEW

File: kernel/base/src/main/java/com/twosigma/beakerx/jvm/object/CyclingOutputContainerLayoutManager.java [hunk 1]
```diff
@@ -15,19 +15,114 @@
...
+
+import java.util.ArrayList;
+import java.util.List;
 
+public class CyclingOutputContainerLayoutManager extends OutputContainerLayoutManager {
+  
+  public static final String PERIOD = "period";
+  
+  private volatile long  period = 5000L;
+  private TimerThread timer = null;
```
Comment (scottdraves) [inline kernel/base/src/main/java/com/twosigma/beakerx/jvm/object/CyclingOutputContainerLayoutManager.java]: is this a leftover?  there should be no timer in java.

File: kernel/base/src/main/java/com/twosigma/beakerx/jvm/object/CyclingOutputContainerLayoutManager.java [hunk 2]
```diff
@@ -15,19 +15,114 @@
...
+      List<Widget> list = new ArrayList<>();
+      list.add(w);
+      WidgetHolder item = new WidgetHolder();
+      item.box = new VBox(list);
+      item.dispalyed = Boolean.FALSE;
+      tabList.add(item);
+    }
+    
+    timer = new TimerThread(InternalVariable.getSimpleEvaluationObject(), tabList );
+    timer.start();
```
Comment (scottdraves) [inline kernel/base/src/main/java/com/twosigma/beakerx/jvm/object/CyclingOutputContainerLayoutManager.java]: no timer in java, please do the animation with JS.

File: kernel/base/src/main/java/com/twosigma/beakerx/widgets/box/CyclingDisplayBox.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
+package com.twosigma.beakerx.widgets.box;
```
Comment (scottdraves) [inline kernel/base/src/main/java/com/twosigma/beakerx/widgets/box/CyclingDisplayBox.java:16]: banner

File: kernel/base/src/main/java/com/twosigma/beakerx/widgets/Layout.java [hunk 4]
```diff
@@ -70,7 +70,6 @@ public Layout() {
     content.put("top", "");
     content.put("visibility", "");
     content.put(WIDTH, getWidth());
-
```
Comment (scottdraves) [inline kernel/base/src/main/java/com/twosigma/beakerx/widgets/Layout.java]: no random little changes

File: kernel/base/src/main/java/com/twosigma/beakerx/widgets/box/CyclingDisplayBox.java (hunk 3 above)
Comment (konstantinTarletski) [reply kernel/base/src/main/java/com/twosigma/beakerx/widgets/box/CyclingDisplayBox.java:16]: Fixed

File: kernel/base/src/main/java/com/twosigma/beakerx/widgets/Layout.java (hunk 4 above)
Comment (konstantinTarletski) [reply kernel/base/src/main/java/com/twosigma/beakerx/widgets/Layout.java]: removed this chage

File: kernel/base/src/main/java/com/twosigma/beakerx/jvm/object/CyclingOutputContainerLayoutManager.java (hunk 1 above)
Comment (konstantinTarletski) [reply kernel/base/src/main/java/com/twosigma/beakerx/jvm/object/CyclingOutputContainerLayoutManager.java]: garbage removed

File: kernel/base/src/main/java/com/twosigma/beakerx/jvm/object/CyclingOutputContainerLayoutManager.java (hunk 2 above)
Comment (konstantinTarletski) [reply kernel/base/src/main/java/com/twosigma/beakerx/jvm/object/CyclingOutputContainerLayoutManager.java]: garbage removed

**Pull request merged**